salt/utils/schedule.py

```diff
@@ -1564,45 +1564,49 @@ def _handle_until(data):
                            'by {0} seconds)'.format(abs(seconds))
 
             try:
-                # Job is disabled, continue
-                if 'enabled' in data and not data['enabled']:
-                    log.debug('Job: %s is disabled', job_name)
-                    data['_skip_reason'] = 'disabled'
-                    data['_skipped_time'] = now
-                    data['_skipped'] = True
-                    continue
-
-                if 'jid_include' not in data or data['jid_include']:
-                    data['jid_include'] = True
-                    log.debug('schedule: Job %s was scheduled with jid_include, '
-                              'adding to cache (jid_include defaults to True)',
-                              job_name)
-                    if 'maxrunning' in data:
-                        log.debug('schedule: Job %s was scheduled with a max '
-                                  'number of %s', job_name, data['maxrunning'])
-                    else:
-                        log.info('schedule: maxrunning parameter was not specified for '
-                                 'job %s, defaulting to 1.', job_name)
-                        data['maxrunning'] = 1
-
-                if not self.standalone:
-                    data['run'] = run
-                    data = self._check_max_running(func,
-                                                   data,
-                                                   self.opts,
-                                                   now)
-                    run = data['run']
+                if run:
+                    # Job is disabled, continue
```

Comment:
I think I worked around this so I don't think it will break the enterprise scheduler but it does mean we could probably simplify some code now. I'll test it out today and report back.